app/components/top-nav-bar.jsx

```diff
@@ -0,0 +1,229 @@
+// https://github.com/EnCiv/civil-pursuit/issues/52
+
+'use strict'
+
+import React, { useState } from 'react'
+import cx from 'classnames'
+import { createUseStyles } from 'react-jss'
+import SvgEncivBlack from '../svgr/enciv-black'
+
+// Placeholder for the user-or-signup component
+const UserOrSignupPlaceholder = () => {
+    return <div>Hello, Andrew! Logout</div>;
+};
+
+
+const TopNavBar = (props) => {
+    const { className, menu, ...otherProps } = props;
+    const [isExpanded, setIsExpanded] = useState(false);
+    const [selectedItem, setSelectedItem] = useState(null);
```

Comment:
You paid more attention than me and saw the underline that remains after you click on something.  To support that we will need to add a defaultValue prop and use that as the initial state for selectedItem.